pkg/controllers/node/termination/terminator/suite_test.go

```diff
@@ -127,53 +129,33 @@ var _ = Describe("Eviction/Queue", func() {
 				Labels:         testLabels,
 				MaxUnavailable: &intstr.IntOrString{IntVal: 1},
 			})
-			ExpectApplied(ctx, env.Client, pod)
-			Expect(queue.Evict(ctx, terminator.NewQueueKey(pod, node.Spec.ProviderID))).To(BeTrue())
+			ExpectApplied(ctx, env.Client, pod, node)
+			Expect(queue.Evict(ctx, pod)).To(BeNil())
```

Comment:
```suggestion
			Expect(queue.Evict(ctx, pod)).To(Succeed())
```
nit: This reads a little better -- you could change all the places you have `BeNil()` to this

Reply:
You resolved this but I didn't get it you did or didn't want to change it